perf_test/scripts/renderer.py

```diff
@@ -0,0 +1,280 @@
+"""This module explodes the JSON configuration file into a flattened list
```

Comment:
This file also came with a small set of unit tests as well. Not that this is _super_ important code to test but I am interested in learning how CI works in this gh org if we wanna grab those and set them up to run on merge or something

Reply:
I tested out in my own repo [here](https://github.com/tedhtchang/kserve-perf/runs/5426470297?check_suite_focus=true)
I think we may add the flowing to this PR under `.github/workflows/unit-test.yml`:
```
name: k6 Unit Test

on:
  pull_request:
    branches:
      - main
    paths-ignore:
      - 'docs/**'
      - '**.md'
      - '.github/**'
      - 'multi_model_tester/**'

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Install Python 3
        uses: actions/setup-python@v1
        with:
          python-version: 3.9.10
      - name: Install dependencies
        run: |
          python -m pip install --upgrade pip
          pip install -r perf_test/requirements.txt
      - name: Run k6 unit tests
        run: python3 -m unittest unit_test/k6/test_renderer.py
```

Reply:
Perfect! This works great!